app/src/component/progressBar/view.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+
+const defaultHeight = 5;
+const defaultBorderRadius = 5;
+const minProgress = 0;
+const maxProgress = 100;
+
+class ProgressBar extends React.PureComponent {
+  static propTypes = {
+    borderRadius: PropTypes.number,
+    color: PropTypes.string.isRequired,
+    height: PropTypes.number,
+    progress: function(props, propName, componentName) {
+      const value = parseInt(props[propName], 10);
+      if (isNaN(value) || props[propName] < minProgress || props[propName] > maxProgress) {
+        return new Error('progress should be between 0 and 100');
```

Comment:
Should this return an error or throw one?

Reply:
According to https://reactjs.org/docs/typechecking-with-proptypes.html, return.